Tangem/App/Services/SwappingFeeFormatter/CommonSwappingFeeFormatter.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  CommonSwappingFeeFormatter.swift
+//  Tangem
+//
+//  Created by Sergey Balashov on 18.05.2023.
+//  Copyright © 2023 Tangem AG. All rights reserved.
+//
+
+import Foundation
+import TangemSwapping
+
+struct CommonSwappingFeeFormatter {
+    private let fiatRatesProvider: FiatRatesProviding
+
+    init(fiatRatesProvider: FiatRatesProviding) {
+        self.fiatRatesProvider = fiatRatesProvider
+    }
+}
+
+// MARK: - SwappingFeeFormatter
+
+extension CommonSwappingFeeFormatter: SwappingFeeFormatter {
+    func format(fee: Decimal, blockchain: SwappingBlockchain) async throws -> String {
+        let fiatFee = try await fiatRatesProvider.getFiat(for: blockchain, amount: fee)
+
+        let feeFormatted = fee.groupedFormatted()
+        let fiatFeeFormatted = await fiatFee.currencyFormatted(code: AppSettings.shared.selectedCurrencyCode)
```

Comment:
Вроде ничего не ждем

```suggestion
        let fiatFeeFormatted = fiatFee.currencyFormatted(code: AppSettings.shared.selectedCurrencyCode)
```

Reply:
Пробовал тоже )
`AppSettings.shared.selectedCurrencyCode` ждем )
не знаю почему, может из за wrapper'a

![Screenshot 2023-05-22 at 11 54 49](https://github.com/tangem/tangem-app-ios/assets/25799680/b5e74a7e-4ed3-4906-b830-3bacc380b094)

Reply:
Подчеркивает fiatFee

Reply:
Окей